cumulusci/tasks/preflight/sobjects.py

```diff
@@ -17,6 +18,48 @@ def _run_task(self):
         )
 
 
+class CheckSObjectPerms(BaseSalesforceApiTask):
+    task_options = {
+        "permissions": {
+            "description": "The object permissions to check. Each key should be an sObject API name, under which Boolean describe values can be specified, "
+            "for example, Account: queryable: True. The output is True if all sObjects and permissions are present and matching the specification.",
+            "required": True
+        }
+    }
+
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+
+        if type(self.options["permissions"]) is not dict:
```

Comment:
This will raise a KeyError if `permissions` is omitted (validation of required permissions doesn't happen until after _init_options)

Reply:
Whoops; corrected.